### PR TITLE
Markdown link extension

### PR DESCRIPTION
**Pull request by dlip**
ref #203

### CODE REVIEW

**Comment by softinio** [comment]
Whats the delay in merging this PR? This is really needed for markdown users. Thanks!

**Comment by fladd** [comment]
Is this project still active?

**Comment by Nudin** [comment]
> Is this project still active?

There's only one maintainer, which apparently has a lack of time at the moment. See #363 – so I guess we'll have to wait for him to return hopefully soon.

**Comment by fladd** [comment]
I see, thanks for the info.

I am currently exploring if we can workaround the Markdown issues for now by having a hook to the VimWiki functions in our .vimrc...

**Comment by mjs2600** [comment]
@fladd did you ever find anything?

**Comment by fladd** [comment]
@mjs2600 No, unfortunately not.

File: syntax/vimwiki_markdown_custom.vim [hunk 1]
```diff
@@ -130,10 +130,15 @@ let g:vimwiki_rxWeblinkMatchDescr0 = g:vimwiki_rxWeblinkMatchDescr
 " LINKS: Setup weblink1 regexps {{{
 let g:vimwiki_rxWeblink1Prefix = '['
 let g:vimwiki_rxWeblink1Suffix = ')'
+let g:vimwiki_rxWeblink1Ext = ''
+if g:vimwiki_markdown_link_include_extension
+  let g:vimwiki_rxWeblink1Ext = '.md'
```
Comment (beuerle) [inline syntax/vimwiki_markdown_custom.vim]: the used markdown extension is defined in `vimrc`. this line should be:

```vim
let g:vimwiki_rxWeblink1Ext = vimwiki#vars#get_wikilocal('ext')
```

**Comment by hq6** [comment]
I am closing this PR since it is superceded by #529, which is being actively worked on.

**Pull request closed without merging**